### PR TITLE
Fix capitalization typo error which causes runtime error in some *nix machines

### PR DESCRIPTION
**Pull request by iamake**
Below is an error on my environment. Patching this PR would fix the problem.

```
[ec2-user@ip-172-26-7-xxx xxx-graph]$ uname -a
Linux ip-172-26-7-xxx 4.9.76-3.78.amzn1.x86_64 #1 SMP Fri Jan 12 19:51:35 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
[ec2-user@ip-172-26-7-xxx xxx-graph]$ ./pm2-app.sh 
module.js:559
    throw err;
    ^

Error: Cannot find module './modules/Postmeta/connectors'
    at Function.Module._resolveFilename (module.js:557:15)
    at Function.Module._load (module.js:484:25)
    at Module.require (module.js:606:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/...xxx.../node_modules/wordexpress-schema/lib/db.js:39:20)
```

### CODE REVIEW

File: src/db.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import {TermModel, TermRelationshipModel, TermTaxonomyModel} from './modules/Ter
 
 // CONNECTORS
 import PostConnectors from './modules/Post/connectors'
-import PostmetaConnectors from './modules/Postmeta/connectors'
+import PostmetaConnectors from './modules/PostMeta/connectors'
```
Comment (ramsaylanier) [inline src/db.js]: I think for this we should just rename the directory to `Postmeta` to be consistent.

**Review by ramsaylanier — CHANGES_REQUESTED** [review]

**Comment by ramsaylanier** [comment]
Also it looks like some commits from the previous PR are still in here. Did you pull from the upstream before submitting this PR?

**Comment by iamake** [comment]
Will close this for another one

**Comment by iamake** [comment]
#22 created to replace this one

**Pull request closed without merging**